test/views/auth/UnlockWalletView/UnlockWalletView.test.tsx

```diff
@@ -8,6 +8,22 @@ import renderSnapshot from '../../../renderSnapshot';
 
 jest.mock('../../../../app/hooks/useMobileCoinD');
 
+jest.mock(
+  'electron',
+  () => {
+    const mockElectron = {
+      app: { getPath: jest.fn(), getVersion: jest.fn() },
+      ipcRenderer: {
+        sendSync: () => {
+          return [];
```

Comment:
No need for braces

Reply:
Prettier/ESLint isn't liking it without the braces here (or other like this)